content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
+- Grafana displays all logs within the log file of the sample application. ((As a noob, I have no idea what I am looking at. You might want to explain what the graph means and what those lines of text at the bottom mean.))
 
-Not only does Loki let you filter logs based on labels, but on specific occurrences:
+Not only does Loki let you filter logs based on labels, but on specific occurrences.
 
-- Filter log lines based on a substring:
+- Enter the following query to filter log lines based on a substring:
 
 ```
 {filename="/var/log/tns-app.log"} |= "error"
 ```
-
+((I got no results from this query until I clicked **Scan for older logs**.))
 Grafana only shows logs within the current time interval. This lets you narrow down your logs to a certain time.
 
-- Click and drag over a log occurrence in the graph to filter logs based on time.
+- Click and drag over a log occurrence in the graph to filter logs based on time. ((Not clear what a log occurrence is. Perhaps you mean graph data?))
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Build a dashboard" %}}
 
-A dashboard gives you an at-a-glance view of your data and lets you track metrics through different visualizations.
+A _dashboard_*_ gives you an at-a-glance view of your data and lets you track metrics through different visualizations.
 
-Dashboards are made up of _panels_, each representing a part of the story you want your dashboard to tell.
+Dashboards are made up _panels_, each representing a part of the story you want your dashboard to tell.
+((MARCUS - Feels like we should have had this definition earlier, when we were exploring the first panel.))
 
-Every panel consists by a _query_ and a _visualization_. The query defines _what_ data you want to display, whereas the visualization defines _how_ the data is displayed.
+Every panel consists by a _query_ and a _visualization_. The query defines _what_ data you want to display, whereas the visualization defines _how_ the data is displayed. ((Feels like we should have had this definition earlier, when we were exploring the first panel.))
 
-- In the side bar, click **Create** to create a new dashboard.
-- Click **Add query**, and enter the query from earlier:
+- In the side bar, hover your cursor over the **Create** (plus sign) icon and then click **Dashboard**.
+- Click **Add query**. 
+- In the **Query editor**, enter the query from earlier and then press Enter:
 
 ```
 sum(rate(tns_request_duration_seconds_count[5m])) by(route)
 ```
 
-- In the **Legend** box, enter "{{route}}" to rename the time series in the legend.
-- Click the **General** panel tab, and change the title to "Traffic".
-- Click the arrow in the top-left corner to go back to the dashboard view.
-- Click the **Save dashboard** icon at the top of the dashboard, to save your dashboard.
+- In the **Legend** field, enter `{{route}}` to rename the time series in the legend. The graph legend updates when you click outside the field.
```

Comment:
Why backticks, `{{route}}`, and not quotation marks "{{route}}"? Because it's code? If so, then we should add this to the style guide.

Reply:
No, it is user input, so should be bold.